### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by Ghilain**
Awesome book lis

- Add index.html page
- Add index.js file
- Add style.css file
- Add README file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+if (addBooks) {
+  books = JSON.parse(addBooks);
+} else {
+  const bookData = [
+    {
+      id: 1,
+      name: 'Lorem ipsum',
+      author: 'Testeroo Testyy',
+    },
+  ];
```
Comment (micheaol) [inline index.js]: Please ensure that the hard-coded book is removed from your code. As you are to generate the data dynamically.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  localStorage.setItem('adbooks', JSON.stringify(bookData));
+  books = JSON.parse(addBooks);
+  window.location.reload();
```
Comment (micheaol) [inline index.js]: Please note that at the moment your books are not been saved in the local storage. When I refresh the browser all the books are removed from the local storage. Kindly fix that according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Ghilain , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js (hunk 1 above)
Comment (Ghilain) [reply index.js]: Done

File: index.js (hunk 2 above)
Comment (Ghilain) [reply index.js]: Done

**Review by Meltrust — APPROVED** [review]
Hi @Ghilain,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the local storage, and for removing the hard coded book, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**